test/integration/framework/master_utils.go

```diff
@@ -204,6 +204,9 @@ func startMasterOrDie(masterConfig *controlplane.Config, incomingServer *httptes
 		)
 	}
 
+	if masterConfig.ExtraConfig.ServiceIPRange.IP == nil {
+		masterConfig.ExtraConfig.ServiceIPRange = net.IPNet{IP: net.ParseIP("10.0.0.0"), Mask: net.CIDRMask(24, 32)}
+	}
```

Comment:
the change seems fine to me:
https://github.com/kubernetes/kubernetes/blob/3a79bb68bd15903effbdb486619637e9598df5e3/pkg/kubeapiserver/options/options.go#L29
https://github.com/kubernetes/kubernetes/blob/b46612a74224b0871a97dae819f5fb3a1763d0b9/pkg/controlplane/services.go#L37